fe/fe-core/src/main/java/com/starrocks/external/hive/HiveColumnStats.java

```diff
@@ -25,6 +25,11 @@
     public HiveColumnStats() {
     }
 
+    public boolean isDefaultValue() {
+        return avgSize == -0.1f && numNulls == -1L && numDistinctValues == -1L && minValue == Double.NEGATIVE_INFINITY
```

Comment:
Could there be a field 'bool unknwon' to represent the stats is unknown?

Reply:
Yes, I think we  should add an Unknown type later，and set unknown if we can not get column statsitics from meta

Reply:
Done